docs/features/for_loops.md

```diff
@@ -0,0 +1,25 @@
+# For Loops
+
+```js
+// Simple for loop that prints out 0 though 9
+// for loops take in a name and an optional type and a list value and will iterate thought the list value
+
+for (i in range(0, 10, 1)) { // range takes in a lower bound (inclusive), and upperbound (exclusive), and a step size are returns an int array
+	print(i)
+} // This will print 0 - 9
+
+/*
+for i 10 {
+	print(i)
+} // deprecated
+*/
```

Comment:
I don't think this needs to be commented out

Reply:
ehhhhh, I just did that because then the output is cleaner.